recipes/dagster-cloud-cli/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "dagster-cloud-cli" %}
+{% set version = "1.7.10" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/dagster-cloud-cli-{{ version }}.tar.gz
+  sha256: 0ff9f243f83d76a110a96027489790e486fe8eaf5b7fca39322fad8fd2ba71b5
+
+build:
+  entry_points:
+    - dagster-cloud = dagster_cloud_cli.entrypoint:app
+  #noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+  skip: true  # [win]
```

Comment:
It is OK for noarch package to fails on Windows with a missing dependency. If that is the case here, we prefer noarch b/c it is faster to build and the package will be ready when the missing dependency is available on Windows.

Reply:
ok - I just rolled back CI should be failing again on windows soon

Reply:
changes are implemented - how can we move forward?